otter/worker/_rcv3.py

```diff
@@ -27,6 +28,11 @@ def _generic_rcv3_request(step_class, request_bag, lb_id, server_id):
         request has no body.
     """
     effect = step_class(lb_node_pairs=[(lb_id, server_id)])._bare_effect()
+
+    svc_req = effect.intent
```

Comment:
If we pop this off of both, won't then deleting fail if the node had already been deleted?